supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -601,6 +601,24 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*:
 
+[discrete]
+[[sos-report]]
+==== image:images/yes.png[yes] sos report (noun)
+*Description*: An _sos report_ is an archive of system information, including configuration and diagnostic information.
+Red{nbsp}Hat Support engineers review sos reports when troubleshooting system problems.
+The command to generate an sos report depends on the RHEL version:
+
+* In RHEL 8 and later, run the `sos report` command (that is, the `sos` command with the `report` argument).
+* In RHEL 7 and earlier, run the `sosreport` command.
+
+NOTE: The `sos` command is an acronym for "son of sysreport", and the official pronunciation is "ess-oh-ess". Therefore, use the indefinite article "an".
```

Comment:
@bergerhoffer @bredamc @bburt-rh This language is mostly fine and a good effort. I would have added this edit directly to the text, but I think we'll need to discuss it here first:

- Question 1. Should we say that if you refer to "sosreport" or "sos report", it should always be written as code in a monospaced font? That's how I usually see it in the docs. We probably need to make that explicit.
- Question 2. OpenShift talks about generating a "sosreport" here: https://docs.openshift.com/container-platform/4.13/support/gathering-cluster-data.html#about-sosreport_gathering-cluster-data How do you know if "sos report" or "sos report" is correct if you're not working in RHEL? Would OpenShift 4.13 for sure be on RHEL8 or is it variable? How do we handle "sos report" in that situation?

Thanks,
Ingrid

Reply:
I updated the PR to split `sos report` and `sosreport` into two separate "Use with caution" entries. 

I also included the word "archive", because I think the main problem area is what to call the file that's created when you run the command, and "archive" indicates that it's an aggregate file.

Let me know what you think!

Reply:
Per the SSG meeting, I think we wanted to update the entries to remove the word "archive" after, does that sound right?

Reply:
Yes, and include a mention of the term that is used for the other RHEL version (in each term entry). It's on my list!